admin/view-translations-post.php

```diff
@@ -34,7 +34,7 @@
 		}
 		?>
 		<tr>
-			<th class = "pll-language-column"><?php echo $language->flag ? $language->flag : esc_html( $language->slug ); // phpcs:ignore WordPress.Security.EscapeOutput ?></th>
+			<th class = "pll-language-column"><?php echo $language->flag ? $language->flag : '<abbr>' . esc_html( $language->slug ) . '</abbr><span class="screen-reader-text">' . esc_html( $language->name ) . '</span>';  // phpcs:ignore WordPress.Security.EscapeOutput ?></th>
```

Comment:
There is no accessibility text for the flag because there is one mentioning the language later in the row. I propose not to add the accessibility text for this abbreviation.